x-pack/plugins/rule_registry/server/utils/create_fetch_start_time.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { PublicContract } from '@kbn/utility-types';
+import { IRuleDataClient } from '../rule_data_client';
+import {
+  ALERT_RULE_UUID,
+  ALERT_INSTANCE_ID,
+  ALERT_STATUS,
+  ALERT_STATUS_ACTIVE,
+  ALERT_START,
+} from '../../common/technical_rule_data_field_names';
+type RuleDataClient = PublicContract<IRuleDataClient>;
+
+export const createFetchStartTime =
```

Comment:
I don't think we need to create a function to query ES to get the `ALERT START`. This info is provided by the executor function itself alongside reason, threshold, etc...
For the inventory https://github.com/fkanout/kibana/blob/270adf49587db4f77dbe3c46f41f92e5d739b6c1/x-pack/plugins/infra/server/lib/alerting/inventory_metric_threshold/inventory_metric_threshold_executor.ts#L89

And it was the same thing for Uptime: https://github.com/elastic/kibana/pull/127478/files#diff-66e5cc62ccd453dfb324f5fa26278cf3755c91caae4150660eef74e0ef52fb9eR452

But if for some reason, we want to get the `ALERT START` from ES, we still don't need to query ES, we only need to expose it via the executor wrapper as it's a recurring function (in the state). https://github.com/fkanout/kibana/blob/270adf49587db4f77dbe3c46f41f92e5d739b6c1/x-pack/plugins/rule_registry/server/utils/create_lifecycle_executor.ts#L294-L295